coresimd/powerpc/altivec.rs

```diff
@@ -394,12 +394,16 @@ mod sealed {
     }
 
     impl VectorMsums<vector_unsigned_int> for vector_unsigned_short {
+        #[inline]
+        #[target_feature(enable = "altivec")]
```

Comment:
the `assert_instr` are missing

Reply:
good call!

Actually looks like the portable instructions do not get mapped correctly.